app/src/main/java/org/brightmindenrichment/street_care/ui/community/data/EventDataAdapter.kt

```diff
@@ -347,12 +347,12 @@ class EventDataAdapter(private val scope: CoroutineScope) {
                         event.eventStartTime = document.get("eventStartTime").toString()
                         event.eventEndTime = document.get("eventEndTime").toString()
                         event.createdAt = document.get("createdAt").toString()
-                        event.helpRequest = document.get("helpRequest") as ArrayList<String> // List<String>
+                        event.helpRequest = (document.get("helpRequest") as? ArrayList<String>) ?: arrayListOf() // List<String>
                         event.helpType = document.get("helpType").toString()
                         event.participants = document.get("participants") as ArrayList<String> // List<String>
                         event.skills = document.get("skills") as ArrayList<String> // List<String>
```

Comment:
add ?: arrayListOf()  to event.participants and event.skills these casts could be nullable